test/api/v3/integration/members/GET-members_id.test.js

```diff
@@ -41,6 +41,7 @@ describe('GET /members/:memberId', () => {
     expect(memberRes.stats.maxMP).to.exists;
     expect(memberRes.stats.maxHealth).to.equal(common.maxHealth);
     expect(memberRes.stats.toNextLevel).to.equal(common.tnl(memberRes.stats.lvl));
+    expect(memberRes.inbox.optOut).to.exists;
```

Comment:
I've pushed a separate commit for the typos and forced the first one to be ok from the start.